dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2/src/org/apache/cxf/transport/http/AbstractHTTPDestination.java

```diff
@@ -251,14 +259,14 @@ public void invoke(final ServletConfig config,
             }
             //Liberty perf change - avoid resize and set init size to 32 and factor 1
             inMessage = new MessageImpl(32, 1);
-            ExchangeImpl exchange = new ExchangeImpl();
-            exchange.setInMessage(inMessage);
             setupMessage(inMessage,
                      config,
                      context,
                      req,
                      resp);
 
+            ExchangeImpl exchange = new ExchangeImpl();
+            exchange.setInMessage(inMessage);
```

Comment:
I don't think there's a need for this change, as it was already happening with the removed code.